std/array.d

```diff
@@ -1475,6 +1512,14 @@ unittest //safety, purity, ctfe ...
     assertCTFEable!dg;
 }
 
+///
+unittest
+{
+	assert(split("hello world") == ["hello","world"]);
+	assert(split("192.168.0.1", ".") == ["192", "168", "0", "1"]);
+	assert(split([1, 2, 3, 4, 5, 1, 2, 3, 4, 5], [2, 3]) == [[1], [4, 5, 1], [4, 5]]);
```

Comment:
This line is a bit long. Consider using a temporary variable or just splitting before the `==`